composeApp/src/commonMain/kotlin/org/ooni/probe/ui/descriptor/add/AddDescriptorScreen.kt

```diff
@@ -131,18 +136,37 @@ fun AddDescriptorScreen(
                         modifier = Modifier.weight(1f),
                     )
                 }
-                LazyColumn {
+            }
+
+            Box(
+                modifier = Modifier
+                    .padding(WindowInsets.navigationBars.asPaddingValues())
+                    .padding(horizontal = 16.dp).fillMaxSize(),
+            ) {
+                LazyColumn(modifier = Modifier.padding(horizontal = 16.dp).padding(bottom = 48.dp)) {
                     items(state.selectableItems) { selectableItem ->
                         TestItem(selectableItem, onChecked = { _ ->
                             onEvent(
                                 AddDescriptorViewModel.Event.SelectableItemClicked(selectableItem),
                             )
                         })
+
+                        if (state.selectableItems.size == 1 && selectableItem.item.test == TestType.WebConnectivity) {
+                            selectableItem.item.inputs.orEmpty().forEach { website ->
```

Comment:
This needs to be a LazyColumn, since lists can have thousands of links. So ideally TestItems and URLs should be sharing the LazyColumn above-